packages/nuxt/src/module.ts

```diff
@@ -20,7 +20,7 @@ export default defineNuxtModule<VuesticOptions>({
 
   defaults: {
     config: {},
-    css: true,
+    css: ['smart-helpers', 'typography'],
```

Comment:
Also, update `@default true` to `@default ['smart-helpers', 'typography']` in types.